src/index.js

```diff
@@ -236,9 +236,9 @@ class Router extends Component {
 }
 
 
-const Route = ({ component:RoutedComponent, url, matches }) => (
-	<RoutedComponent {...{url, matches}} />
-);
+const Route = ({ component: RoutedComponent, url, matches }) => {
+	return h(RoutedComponent, { url, matches });
```

Comment:
Can just call this `component` now. (I don't mind changing after merge if you're good)